lib/helper.py

```diff
@@ -167,6 +167,7 @@ def copy_dir_file(src, dest):
         if os.path.exists(d_file):
             logger.info("file already exist in %s it will going to overwrite with new file" % dest)
         shutil.copy2(s_file, d_file)
+        os.chmod(d_file, 0o755)
```

Comment:
suggest: we can use enums instead of nos.
```
import stat
os.chmod(d_file, stat.S_IRWXU|stat.S_IXUSR|stat.S_IXGRP|stat.S_IXOTH)
```
this would change permission to `-rwx--x--x.`

Reply:
sure, will do that. Was going for this approach for readability mainly.